ui/page/components/treasury_list.go

```diff
@@ -0,0 +1,156 @@
+package components
+
+import (
+	"strings"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+	"gioui.org/unit"
+	"gioui.org/widget"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type TreasuryItem struct {
+	Policy            dcrlibwallet.TreasuryKeyPolicy
+	OptionsRadioGroup *widget.Enum
+	VoteChoices       [3]string
+	SetChoiceButton   decredmaterial.Button
+}
+
+func (t *TreasuryItem) SetVoteChoices(voteChoices [3]string) {
+	t.VoteChoices = voteChoices
+}
+
+func TreasuryItemWidget(gtx C, l *load.Load, treasuryItem *TreasuryItem) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layoutPiKey(gtx, l, treasuryItem.Policy)
+		}),
+		layout.Rigid(layoutVoteChoice(l, treasuryItem)),
+		layout.Rigid(func(gtx C) D {
+			return layoutPolicyVoteAction(gtx, l, treasuryItem)
+		}),
+	)
+}
+
+func layoutPiKey(gtx C, l *load.Load, treasuryKeyPolicy dcrlibwallet.TreasuryKeyPolicy) D {
+
+	statusLabel := l.Theme.Label(values.TextSize14, treasuryKeyPolicy.PiKey)
+	backgroundColor := l.Theme.Color.LightBlue
+
+	return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			lbl := l.Theme.Label(values.TextSize20, values.String(values.StrPiKey))
+			lbl.Font.Weight = text.SemiBold
+			return layout.Flex{}.Layout(gtx,
+				layout.Rigid(lbl.Layout),
+			)
```

Comment:
whats the idea behind adding this extra flex layout?

```suggestion
			return lbl.Layout(gtx)
```